src/Analyzer.Core.UnitTests/TemplateAnalyzerTests.cs

```diff
@@ -43,7 +43,7 @@ public void AnalyzeTemplate_ValidInputValues_ReturnCorrectEvaluations(string res
         }
 
         [DataTestMethod]
-        [DataRow("SimpleNestedFail.bicep", "{ \"SimpleNestedFailModule.bicep\": [5, 16, 20, 21, 27, 31, 32, 33] }", DisplayName = "Simple bicep nested template example")]
+        [DataRow("SimpleNestedFail.bicep", "{ \"SimpleNestedFailModule.bicep\": [16, 20, 21, 27, 31, 32, 33] }", DisplayName = "Simple bicep nested template example")]
```

Comment:
Similar - were the failures on these two lines due to the removed rule?

Reply:
Yes.